front/src/Api/iframe/nav.ts

```diff
@@ -31,7 +32,7 @@ class WorkadventureNavigationCommands extends IframeApiContribution<Workadventur
             "type": 'loadPage',
             "data": {
                 url
-            }
+            } as LoadPageEvent
```

Comment:
its not strictly neccessary to type those objects as sendToWorkadventure in combination with the the event type already implicitly expects the right type